src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -281,3 +281,48 @@ define void @__quantum__qis__z__ctl(%Array* %.ctls, %Qubit* %.q) {
 }
 
 
+;===============================================================================
+; quantum.qis math functions
+;
+
+; LLVM intrinsics (https://llvm.org/docs/LangRef.html):
+declare double      @llvm.sqrt.f64(double %.val)
+;declare double      @llvm.ceil.f64(double %Val)

```

Comment:
Can we drop this from the PR or is this commented to be added back in later? If so, would you mind adding a note on that with optionally a link to a GH ticket?

Reply:
This is work in progress to be added in the next commit/PR.
(I was working on it when I got the first feedback about Sqrt, so I postponed Ceil until I resolve all for Sqrt)